fmlcore/src/main/java/net/minecraftforge/fml/ModLoadingContext.java

```diff
@@ -46,7 +46,10 @@ public String getActiveNamespace() {
      * @param point The extension point to register
      * @param extension An extension operator
      * @param <T> The type signature of the extension operator
+     *
+     * @deprecated take a look at {@link net.minecraftforge.fml.event.lifecycle.FMLConstructModEvent}
```

Comment:
Pointing to the exact method would be a better dev migration UX as it makes it clear the method hasn't changed - only its location. In IntelliJ you can right click on a method -> copy -> copy reference to get the link that'll work with javadocs.
```suggestion
     * @deprecated Use {@link net.minecraftforge.fml.event.lifecycle.FMLConstructModEvent#registerExtensionPoint} instead
```